service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/CustomGcpIamRoleMapping.java

```diff
@@ -46,7 +46,13 @@ public class CustomGcpIamRoleMapping {
           "bigquery.tables.export",
           "bigquery.tables.getData",
           "bigquery.tables.list",
-          "bigquery.tables.get");
+          "bigquery.tables.get",
+          // Permissions of Service Account Token creator, required for bq data transfer service.
+          "iam.serviceAccounts.getAccessToken",
```

Comment:
This block of permissions is granted on a BQ dataset, I don't think the SA related ones will have any effect here